### PR TITLE
[FEATURE] UI - Add CRUD for Roles and RoleBindings resources

### PR DESCRIPTION
**Pull request by Gladorme**
# Description

This PR contains the following:
- Change icon for admin space, previous admin icon used by Role Bindings
- Add CRUD for Role and Global Roles at project level. Global Role and Global RoleBindings will be added in a next PR 😄 
- Role and RoleBinding editor forms are fully validated with Zod

# Screenshots

## Roles
image
image

## RoleBindings
image
image

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [x] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [x] Visual tests are stable and unlikely to be flaky.

### CODE REVIEW

File: ui/plugin-system/src/validation/role.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import { z } from 'zod';
+import { resourceIdValidationSchema } from '@perses-dev/plugin-system';
+
+export const permissionValidationSchema = z.object({
+  // TODO: use SCOPE & ACTIONS constants
+  actions: z.array(z.enum(['*', 'create', 'read', 'update', 'delete'])).nonempty('Must contains at least 1 action'),
+  scopes: z
+    .array(
+      z.enum([
```
Comment (Gladorme) [inline ui/plugin-system/src/validation/role.ts:22]: I don't understand why I can't use PROJECT_SCOPES here or ACTIONS, will check later with GlobalRole & GlobalRoleBinding PR 🤔

File: ui/app/src/model/config-client.ts [hunk 2]
```diff
@@ -143,6 +142,15 @@ export function useIsReadonly() {
   return data.security.readonly;
 }
 
+export function useIsAuthorizationEnabled() {
```
Comment (Nexucis) [inline ui/app/src/model/config-client.ts:145]: with the new context, can you move this method to the context please ?

Comment (Nexucis) [reply ui/app/src/model/config-client.ts:145]: and perhaps use it in the guard

**Review by Nexucis — APPROVED** [review]
minus the changes required by the config provider, looks good to me!

**Pull request merged**